### PR TITLE
🐛 [Story attachment] Opening outlink error for closeButton being null

### PR DESCRIPTION
**Pull request by mszylkowski**
Fixes https://github.com/ampproject/error-reporting/issues/147

Outlinks when opened don't have a `closeButton` that they can change the `tabindex` so it was throwing an error. By checking if there's such button we can prevent the error and run the last lines of the `open()` method properly.

### CODE REVIEW

**Review by calebcordry — COMMENTED** [review]
Can we add a test?

**Comment by mszylkowski** [comment]
For sure, I think in general the feature is undertested but any unit test that checks this functionality should have picked it up. I'll add a generic test for the outlink component that would have caught this error (and potentially many other ones like it).

File: extensions/amp-story-page-attachment/0.1/test/test-amp-story-page-attachment.js [hunk 1]
```diff
@@ -118,4 +129,19 @@ describes.realWin('amp-story-page-attachment', {amp: true}, (env) => {
...
+    anchorEl.setAttribute('target', '_blank');
+
+    const clickSpy = env.sandbox.spy(anchorEl, 'click');
+
+    await outlink.buildCallback();
+    await outlink.layoutCallback();
+
+    outlink.open();
+
+    expect(clickSpy).to.be.called;
```
Comment (calebcordry) [inline extensions/amp-story-page-attachment/0.1/test/test-amp-story-page-attachment.js:144]: nit: `calledOnce`

**Review by calebcordry — APPROVED** [review]
Thanks for adding tests!

**Comment by processprocess** [comment]
Thank you for fixing this & writing a test for it ❤️

**Pull request merged**